Makefile

```diff
@@ -23,3 +23,23 @@ export GOPROXY=https://proxy.golang.org
 .PHONY: docker-build
 docker-build: build
 
+# Helper functions to improve local testing experience
+.PHONY: create-account
+create-account:
+	@oc apply -f myaccount.json
```

Comment:
This file doesn't exist in this PR (it's the one recommended to create as part of our main README), happy to add it once the decision above it made - can potentially add it to the workflow folder too

Reply:
Are planning on designating an AWS account just for testing or will the user need to edit the file? just curious.

Reply:
Yup, this would require the user to edit the file .... actually good point if that is the case, probably better to move it all into a script which can validate the AWS Account ID is set in the json before application